spec/requests/create_work_spec.rb

```diff
@@ -418,6 +424,85 @@
+
+        context 'with only title changed' do
+          let(:title) { 'Added a title' }
+          let(:work_params) do
+            {
+              title:,
+              abstract: '',
+              license: default_license,
+              work_type: 'text',
+              release: 'immediate'
+            }
+          end
+
+          it 'saves the draft work with title modified for change description' do
+            post "/collections/#{collection.id}/works", params: { work: work_params,
+                                                                  commit: 'Save as draft' }
+            expect(response).to have_http_status(:found)
+            work_version = Work.last.head
+            expect(work_version.title).to eq title
+            expect(work_version.license).to eq default_license
+            last_event = Work.last.events.last
+            expect(last_event.event_type).to eq 'update_metadata'
+            expect(last_event.description).to eq 'title of deposit modified'
+          end
+        end
+
+        context 'with default license selected' do
+          let(:work_params) do
+            {
+              title: '',
+              abstract: '',
+              license: default_license,
+              work_type: 'text',
+              release: 'immediate'
+            }
+          end
+
+          it 'saves the draft work with Created for change description' do
+            post "/collections/#{collection.id}/works", params: { work: work_params,
+                                                                  commit: 'Save as draft' }
+            expect(response).to have_http_status(:found)
+            work_version = Work.last.head
+            expect(work_version.license).to eq default_license
+            last_event = Work.last.events.last
+            expect(last_event.event_type).to eq 'update_metadata'
+            expect(last_event.description).to eq 'Created'
+          end
+        end
+
+        context 'with different license selected' do
+          let(:license) { 'Apache-2.0' }
+          let(:work_params) do
+            {
+              title: '',
+              abstract: '',
+              license:,
+              work_type: 'text',
+              release: 'immediate'
+            }
+          end
+
+          it 'saves the draft work with license modified for change description' do
+            post "/collections/#{collection.id}/works", params: { work: work_params,
+                                                                  commit: 'Save as draft' }
+            expect(response).to have_http_status(:found)
+            work_version = Work.last.head
+            expect(work_version.license).to eq license
+            last_event = Work.last.events.last
+            expect(last_event.event_type).to eq 'update_metadata'
+            expect(last_event.description).to eq 'license modified'
+          end
+        end
+      end
+
```

Comment:
new tests that verify that a created work gets the correct event description for a few scenarios (no changes, default license changed, title changed)